docs/categorization-dashboard.md

```diff
@@ -0,0 +1,50 @@
+# Categorization dashboard
+
+To check that the categorization works well, we have tests that generate a dashboard showing global and local models performances.
+
+## Run tests
+
+Since these tests are executed on real data, the data is encrypted. To be able to run the tests, you have to decrypt it:
+
+```
+yarn decrypt-banking-tests
+```
+
+The global categorization model tests also needs the model parameters. These parameters should be downloaded:
```

Comment:
To avoid any question like "why don't you embed the model's parameter in the repo", i would have been slightly more precise : "The global categorization model tests also needs the **current and latest** model parameters